core/App/ehsm_provider.cpp

```diff
@@ -293,6 +293,9 @@ uint32_t EHSM_FFI_CALL(const char *reqJson, char *respJson)
     case EH_VERIFY_QUOTE:
         ffi_verifyQuote(payloadJson, respJson);
         break;
```

Comment:
please combine the 2 commits into one

Reply:
Sure, the commits should be squashed. Please check, thanks.